### PR TITLE
Request to squash commits if the PR has multiple

### PR DESCRIPTION
**Pull request by mandrigin**
Allow our script to do interactive rebase while merging the PR, should streamline the PR process.

PS: if you know a better way to squash stuff non-interactively, feedback appreciated

status: ready

### CODE REVIEW

**Review by rasom — APPROVED** [review]

File: scripts/merge-pr.sh [hunk 1]
```diff
@@ -104,6 +104,13 @@ check_is_pr_single_commit() {
   fi
 }
 
+request_to_squash() {
+  if [[ $(git rev-list $BRANCH..$PR_LOCAL_BRANCH | wc -l) -gt 1 ]] ;then
+    confirm "PR has multiple commits, do interactive rebase?"
+    git rebase -i $BRANCH
```
Comment (jakubgs) [inline scripts/merge-pr.sh:110]: A simpler way would be something like:
```bash
MSG=$(git log --format=%B -n 1 $PR_LOCAL_BRANC)
git reset --soft $BRANCH
git commit -m "$MSG"
```

Comment (jakubgs) [reply scripts/merge-pr.sh:110]: Hm, no, but that would take last commit message, not first commit message...

Comment (mandrigin) [reply scripts/merge-pr.sh:110]: my thought process was that having an interactive rebase is better than nothing :-D later we can improve the flow even more

Comment (jakubgs) [reply scripts/merge-pr.sh:110]: That's fair.

**Review by jakubgs — APPROVED** [review]

File: scripts/merge-pr.sh (hunk 1 above)
Comment (jakubgs) [reply scripts/merge-pr.sh:110]: Though maybe a message or a link to how to use `rebase -i` would be nice.

**Comment by mandrigin** [comment]
it doesn't affect anything except merging multi-commit PRs, so I will merge it with no separate QA. tested it locally and it _works on my machine_ 😆

**Comment by jakubgs** [comment]
but_i_understand

**Pull request merged**